src/test/java/web/tests/FlipkartTest.java

```diff
@@ -216,14 +218,35 @@ public void testPolicyBackToTop() {
      */
     @ParameterizedTest
     @CsvFileSource(resources = "/testProductPageDesign.csv")
-    public void testProductPageDesign(String item,int position){
+    public void testProductPageDesign(String item, int position) {
         ProductPage productPage = new HomePage().popUpCancel().setShoes(item).searchShoes().OpenProductPage(position);
         productPage.scrollToBottom();
-        ArrayList<String> headers  = new ArrayList<>(Arrays.asList(map.get("headers").split(",")));
-        for(String headersInSelectionPage:headers){
+        ArrayList<String> headers = new ArrayList<>(Arrays.asList(map.get("headers").split(",")));
+        for (String headersInSelectionPage : headers) {
             softAssert.assertThat(productPage.isSelectionDisplayed(headersInSelectionPage)).
                     as("Json selected Headers is Not matching with selection page Headers").isTrue();
 
         }
     }
+
+    @Test
+    public void flipkartSocialMedia(){
+        HomePage homePage = new HomePage().popUpCancel();
+        SocialMediaPage socialMediaPage=null;
+        ArrayList<String> socialMediaLinks = new ArrayList<>(Arrays.asList(map.get("links")
+                .split(",")));
+        ArrayList<String> placeholdersOfSocialSites= new ArrayList<>(Arrays.asList(map.get("placeholders")
+                .split(",")));
+        for (int i = 0; i < socialMediaLinks.size(); i++) {
+            logger.info("*****links*****"+socialMediaLinks.get(i));
+            logger.info("*****placeholders*****"+placeholdersOfSocialSites.get(i));
+            socialMediaPage =homePage.clickLink(socialMediaLinks.get(i)).checkPlaceholders(placeholdersOfSocialSites.get(i));
```

Comment:
Use the code which is written inside : checkPlaceholders(placeholdersOfSocialSites.get(i)
in 
getSocialMediaUrl()
and remove that method.